src/views/ManageList.jsx

```diff
@@ -49,13 +38,13 @@ export function ManageList() {
 					/>
 				</div>
 				<select
-					onChange={(e) => setItemDuration(e.target.value)}
+					onChange={(e) => setItemDuration(Number(e.target.value))}
 					name="replaceTime"
 					value={itemDuration}
 				>
-					<option value={'soon'}>Soon</option>
-					<option value={'kindOfSoon'}>Kind of soon</option>
-					<option value={'notSoon'}>Not Soon</option>
+					<option value={'7'}>Soon</option>
```

Comment:
This was tricky but in the issue it has this:

<img width="431" alt="image" src="https://github.com/the-collab-lab/tcl-67-smart-shopping-list/assets/19897674/60f00875-a52c-464d-9f46-c6cc228faa0c">

And was marked as done. I'm not sure if there were values here before you started your issue but that's easy to miss. I would always double check the checked items to see if they were completed just incase.

Reply:
So the labels on the front-end show to the user, but to be honest, there is a question to ask ourselves on whether or not this is too vague to show to the user. Since it's not clear to the user what Soon, Kind of soon, and Not soon means for our app.

Reply:
I was definitely wondering the same thing! Maybe we could do something like heading the buttons with "Remind to repurchase in...", and then add buttons labeled 7 days, 14 days, and 30 days underneath for more clarity